### DISCUSSION TITLE
Updating Zigbee2mqtt with old Home Assistant

### DISCUSSION THREAD

**Discussion by jeppper** [Koenkk/zigbee2mqtt#8343] [General]
Is it possible to update Zigbee2mqtt to the latest version when using it with Home Assistant 0.106.6?
It seems in the release notes that a newer version of HA is needed when updating, but I can´t figure out if this is only necassary when using some special features,

**Comment by Koenkk** [comment]
It is needed due to https://github.com/Koenkk/zigbee2mqtt/pull/7980 and not recommended, all your `energy` sensors will likely break.

**Reply by jeppper** [reply]
What kind of energy sensors are you thinking of? My installation mostly consists of Hue bulbs, Xiaomi movement and temperature sensors and window contacts.
What version is the last "safe" one to use then?
